src/workload.py

```diff
@@ -120,6 +120,9 @@ def generate_password(self) -> str:
 
     @override
     def get_version(self) -> str:
+        if not self.alive:
+            return ""
+
```

Comment:
**question:** Is there harm in having the check as one of the first lines in `healthy()`?

Reply:
I think this would make more sense, as "you cannot be healthy if you are not alive". I tend to avoid modifying existing functions when I add new ones, though

Reply:
Additional followup: `healthy` is used in `_on_pebble_ready` and `_on_cluster_changed`, where `alive` is used just a few lines before. Therefore, I do not think it is necessary to proceed with the change. Do you agree?